guides/common/modules/proc_updating-server.adoc

```diff
@@ -0,0 +1,52 @@
+[id="Updating_Server_{context}"]
+= Updating {ProjectServer}
+
+.Prerequisites
+
+* Ensure that you have synchronized {ProjectServer} repositories for {Project}, {SmartProxy}, and {project-client-name}.
```

Comment:
After a recent BZ we triaged last week, I wonder if we should explicitly name the repos you need to have synced. @ehelms thoughts?

Reply:
@Griffin-Sullivan Can you please share the BZ so we can take a look?

Reply:
@mjivraja I don't see any progress here :)

Reply:
Apologies @Lennonka I haven't received the data yet, nor the BZ. But, I am actively syncing for the same.

Reply:
This syncing action is for updating Smart Proxy / Capsule and is not required to update Foreman / Satellite. I'm curious why we are putting this here as a step and not as part of the Smart Proxy / Capsule steps? If a user wants to just update their Foreman / Satellite this step is unnecessary.